i18n/pt-br.yaml

```diff
@@ -0,0 +1,93 @@
+# General
+- id: read_more
+  translation: "Leia mais…"
+
+- id: menu_label
+  translation: "Menu"
+
+- id: sidemenu_title
+  translation: "Menu"
+
+# Post meta
+- id: meta_lastmod
+  translation: "Última modificação"
+
+- id: meta_translations
+  translation: "Traduções"
+
+# Table of Contents
+- id: toc_title
+  translation: "Conteúdo da página"
+
+# Post nav
+- id: post_nav_prev
+  translation: "Anterior"
+
+- id: post_nav_next
+  translation: "Próximo"
+
+# Authorbox
+- id: authorbox_name
+  translation: "Sobre {{ .Count }}"
+
+# Sidebar
+- id: sidebar_warning
+  translation: "ATENÇÃO"
+
+- id: sidebar_recommendation
+  translation: "Ative pelo menos um widget da barra lateral."
+
+# Search widget
+- id: search_placeholder
+  translation: "BUSCAR..."
+
+# Languages widget
+- id: languages_title
+  translation: "Línguas"
+
+# Categories widget
+- id: categories_title
+  translation: "Categorias"
+
+# Recent Posts widget
+- id: recent_title
+  translation: "Postagens recentes"
+
+# Social widget
+- id: social_title
+  translation: "Social"
+
+# Tags List widget
+- id: tags_title
+  translation: "Tags"
+
+# Footer
+- id: footer_credits
+  translation:
+    "Gerado com <a href=\"https://gohugo.io/\" rel=\"nofollow noopener\" target=\"_blank\">Hugo</a> com o tema \
+    <a href=\"https://github.com/Vimux/Mainroad/\" rel=\"nofollow noopener\" target=\"_blank\">Mainroad</a>."
+
+# 404
+- id: 404_title
+  translation: "404. Página não encontrada"
+
+- id: 404_text
+  translation:
+    "A página que você estava procurando parece ter sido excluída ou não existe. Por favor, use a pesquisa ou vá para"
+
+- id: 404_linktext
+  translation: "página principal"
+
+# No posts empty state
+- id: noposts_warning_title
+  translation: "Você ainda não tem nenhuma postagem!"
+
+- id: noposts_warning_description
+  translation:
+    "<p>Quando você criar uma postagem em qualquer pasta (seção) dentro do diretório content, ele vai aparecer aqui.</p> \
+    <p>Apenas uma seção (a que conter mais postagens) será exibida na página principal por padrão.</p>"
```

Comment:
There is no need to use `p` tags here.

Reply:
Thank you for review. I made changes requested :)